pages/1.10/security/ent/perms-reference/index.md

```diff
@@ -97,13 +128,23 @@ By convention, `full` indicates that the permission supports all other action id
 
 ## <a name="secrets"></a>Secret Store Permissions
 
+These permissions control access to the [Secrets API](/1.10/security/ent/secrets/secrets-api/). A Mesos framework must have
+permission granted to its DC/OS service account in order to access a given secret. If you are looking for information on how to launch
+Marathon applications using secrets see [Configuring services and pods to use secrets](/1.10/security/ent/secrets/use-secrets/).
+
 |                                                                                                                                 Permission string                                                                                                                                 | full | C | R | U | D |
 |-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------|---|---|---|---|
 | `dcos:secrets:default:[<path-name>/]<secret-name>`<br> Controls access to individual [secrets](/1.10/security/ent/secrets/).                                                                                                                                                           | x    | x | x | x | x |
 | `dcos:secrets:list:default:/[<path>]`<br> Controls view access to the names of [secrets](/1.10/security/ent/secrets/).                                                                                                                                                                 |      |   | x |   |   |
 
 ## <a name="superuser"></a>Superuser Permissions
 
+Similar to the Windows `Administrator` or Linux `root` accounts, DC/OS has a
+`superuser` role. A user with permission to perform the `full` action on the
```

Comment:
Let's not use the term 'role' here. Maybe 'concept'?

Also:

> The special permission dcos:superuser (action identifier irrelevant) overrides everything else: a DC/OS authorizer must never deny an action if this permission is set.

(https://wiki.mesosphere.com/pages/viewpage.action?pageId=113019009)

(`full` is not required as of that spec)

Reply:
A combination of the original and the updated:
```
Similar to the Windows `Administrator` or Linux `root` accounts, DC/OS has the concept of the `superuser`.
```

Reply:
@jgehrcke I've changed the wording to match what @jongiddy suggested.
I've also changed the `dcos:superuser` permission's columns to have 'x's in all columns and updated the description of what actions are allowed, as of the wiki page you referenced.